src/sql/src/plan/statement/ddl.rs

```diff
@@ -1278,30 +1278,30 @@ fn kafka_sink_builder(
         Some(_) => bail!("consistency_topic must be a string"),
     };
 
-    let exactly_once = match with_options.remove("exactly_once") {
+    let reuse_topic = match with_options.remove("reuse_topic") {
         Some(Value::Boolean(b)) => b,
         None => false,
-        Some(_) => bail!("exactly-once must be a boolean"),
+        Some(_) => bail!("reuse_topic must be a boolean"),
     };
 
-    if exactly_once && consistency_topic.is_none() {
-        bail!("exactly-once requires a consistency topic");
+    if reuse_topic && consistency_topic.is_none() {
+        bail!("reuse_topic requires a consistency topic");
     }
 
-    let transitive_source_dependencies: Vec<_> = if exactly_once {
+    let transitive_source_dependencies: Vec<_> = if reuse_topic {
         for item in root_dependencies.iter() {
             if item.item_type() == CatalogItemType::Source {
                 if !item.source_connector()?.yields_stable_input() {
                     bail!(
-                    "all input sources of an exactly-once Kafka sink must be replayable, {} is not",
+                    "reuse_topic makes a sink an exactly-once sink, all input sources of an exactly-once Kafka sink must be replayable, {} is not",
```

Comment:
I think we should reword this because this error message begs the question "well wtf is an exactly-once sink?"

Maybe we can reword as:

"reuse_topic requires that sink input dependencies are replayable, {} is not"

Reply:
👍